.github/workflows/prover.yml

```diff
@@ -1,7 +1,8 @@
 name: Run prover guest code checks
 
 on:
-  pull_request:
+  pull_request_target:
+    types: [opened, synchronize, reopened, edited]
```

Comment:
Can we work around this?
`zizmor` is flagging this as HIGHLY insecure. See https://woodruffw.github.io/zizmor/audits/#remediation_1